src/planning/plan/plan-consumer.ts

```diff
@@ -93,8 +93,8 @@ export class PlanConsumer {
       const usesHandlesFromActiveRecipe = suggestion.plan.handles.find(handle => {
         // TODO(mmandlis): find a generic way to exlude system handles (eg Theme),
         // either by tagging or by exploring connection directions etc.
-        return !!handle.id &&
-               !!this.arc.activeRecipe.handles.find(activeHandle => activeHandle.id === handle.id);
+        return Boolean(handle.id) &&
```

Comment:
Note: use != rather than !== for null checks so we don't change behavior (Boolean(undefined) is falsy, undefined == null, undefined !== null)